@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/core/fn_coreUpdate.sqf

```diff
@@ -24,7 +24,7 @@ if (isGamePaused) exitwith {};
 private _config      = configFile >> "CfgVehicles" >> typeof _heli;
 private _flightModel = getText (_config >> "fza_flightModel");
 
-private _deltaTime   = ((["sfmplus_deltaTime"] call BIS_fnc_deltaTime) min 1/30);
+private _deltaTime   = ["sfmplus_deltaTime"] call BIS_fnc_deltaTime;
```

Comment:
This looks like an unrelated change?

Reply:
This is the correct implementation of deltaTime to ensure the holds and simulation run smoothly, which was required for testing. It's not unrelated.